x/staking/keeper/historical_info.go

```diff
@@ -2,9 +2,8 @@ package keeper
 
 import (
 	"context"
-
+	consensusv1 "cosmossdk.io/api/cosmos/consensus/v1"
```

Comment:
since the hybrid router and proto v2 is deprecated in server v2 these should be gogo type imports right?